release-team/role-handbooks/bug-triage/README.md

```diff
@@ -1,195 +1,223 @@
+  * `is:pr "1.14.0-beta"' [PRs mentioning beta version](https://github.com/kubernetes/kubernetes/pulls?utf8=%E2%9C%93&q=%221.14.0-beta%22)
 
-We're still doing the red/yellow/green reports:
+### Reports [Freeze]
 
-* red: major breakage issues which are not making daily progress, and enhancement issues without a PR which looks to be approved soon.
-* yellow: major breakage issues with a PR in good shape, minor breakage issues, and enhancement issues with a PR in good shape.
-* green: minor breakage issues with a good PR and enhancement issues with a PR that is just waiting on merge.
+A few days after Code Freeze kicks in, daily 'burndown' meetings start, which means that either you or a shadow has to give updates every day. See [Issue Categorization](#issue-categorization).
 
 There will also be several enhancements which will have requested exceptions from the normal release timeline/requirements.  You'll want to track the exceptions specifically and report on them.  It is useful to be tracking in a spreadsheet the open issues/PRs which have caught your attention, because GitHub queries are only point in time.  It's easy for an issue you want to follow to stop showing up in a query and thus fall off your radar if you're only going by the queries.
 
-The Release Lead will include a template for this information in the Monday/Wednesday/Friday release team meeting notes.
-
 ## Code Thaw
 
-Starting a few working days before the first Release Candidate, we go into Sudden Death Overtime.  At this point, you need to make sure that three things happen:
+Starting a few working days before the first Release Candidate, Code Freeze labeling restrictions are lifted.  At this point, you need to make sure that three things happen:
 
 1. major breakage bugs get fixed immediately
 2. any pending PRs get approved and merged
-3. anything else gets kicked out of the release
+3. *anything else gets kicked out of the release*
 
-During this period, it's reasonable to expect issue owners and SIG leads to get back to you within hours (check their time zones, though).  In cases where SIG Leads are unavailable, you may need to appeal to Kubernetes project leaders to deal with stuck PRs.
+During this period, it's reasonable to expect issue owners and SIG leads to get back to you within hours (check their time zones, though). In cases where SIG Leads are unavailable, you may need to appeal to Kubernetes project leaders to deal with stuck PRs.
 
 Another part of staying on top of code churn, regression, and risk is monitoring commits to master and the release branch especially in the final days of the release, as shown below in Sample Searches for this phase.  On occasion things will merge that are unexpected by the release team.  This possibility needs monitored and when it happens the commits need triaged for destabilizing risk and proper exception justification, tests, docs, etc.
```

Comment:
since we're here: "...this possibility needs to be monitored..."

Reply:
similar: "...the commits need to be triaged..."

Reply:
fixed, missed that paragraph